zypper-docker.go

```diff
@@ -25,6 +25,7 @@ var exitWithCode func(code int)
 var killChannel chan bool
 
 func main() {
+	os.Args = fixArgsForZypper(os.Args)
```

Comment:
nitpicking: do this just before calling newApp, since this is related to the creation of the app.